cypress/e2e/3-actions.js

```diff
@@ -53,4 +58,287 @@ describe('Colony can mint tokens via action', () => {
       });
     });
   });
+  it('Can make payment', () => {
+    const amountToPay = 10;
+    const annotationText = 'Test annotation';
+    const accounts = Object.entries(ganacheAccounts.private_keys).map(
+      ([address]) => address,
+    );
+    const cutAddress = splitAddress(createAddress(accounts[1]));
+    const paidAmount = new Decimal(amountToPay).sub(
+      new Decimal(1.0001).div(100).mul(amountToPay),
+    );
+    let prevColonyFunds;
+
+    cy.login();
+
+    cy.visit(`/colony/${Cypress.config().colony.name}`);
+
+    cy.getBySel('colonyTotalFunds', { timeout: 30000 }).then(($text) => {
+      const amount = $text.text().split(',').join('');
+      prevColonyFunds = amount;
+    });
+
+    cy.getBySel('newActionButton', { timeout: 60000 }).click();
+    cy.getBySel('indexModalItem').eq(0).click();
```

Comment:
I can agree with that, because as you mentioned, in the future the order might change. Gonna wait a bit to see what the others think about it before implementing it or not.